docker/indexer/proto/example.textproto

```diff
@@ -0,0 +1,15 @@
+# proto-message: Repository
+
+address: "https://github.com/protocolbuffers/protobuf.git"
+name: "protobuf"
+type: GIT
+base_cpe: "cpe:2.3:a:google:protobuf:"
+version_regex: "v({\\d}+\\.{\\d}+)"
```

Comment:
Would it work at all to just have more generic logic to try to pull these out? e.g. we can detect runs of digits and extract them. 

We have some similar logic implemented in Python here: https://github.com/google/osv.dev/blob/e84e63b9b8ae1b89992ec5ca750a4d2ae6175459/osv/bug.py#L31

There might be inconsistent formatting even within the same repo, and a more general approach might help and reduce manual effort when we want to scale this up.

Reply:
I added the generic version regex, which is used if the specific one fails.